addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js

```diff
@@ -178,7 +178,9 @@ odoo.define('point_of_sale.PaymentScreen', function (require) {
                     syncedOrderBackendIds = await this.env.pos.push_single_order(this.currentOrder);
                 }
             } catch (error) {
-                this.error = true;
+                //  exception for "XmlHttpRequestError"
+                if (error.code != -32098)
```

Comment:
```suggestion
                if (error.code == 700)
```
Hi @EsamHussein. Thank you so much for this PR. Can you consider my suggestion? I asked the author and the main reason why `this.error` is there is because of the `code = 700`. So perhaps we can be more specific on the if condition.

Also, can you reformat the commit description to:
```
[FIX] point_of_sale: receipt printing available for offline mode
```

To do all my suggestions, you can modify the code locally (in your machine) taking into account my suggestion, then you amend the commit message, then you force push the branch. :)

Reply:
@caburj 
Done

Reply:
Sorry but this fix can be represented by a single commit. Can you squash the commits into 1?

Reply:
I closed this request and make a new one, with one commit
https://github.com/odoo/odoo/pull/72240
because I get some issues with branch rebase 

Thanks